unconstrained_test.go

```diff
@@ -1503,16 +623,5 @@ func testMinimize(t *testing.T, tests []UnconstrainedTest, method Method) {
 				}
 			}
 		}
-
```

Comment:
An example is the commented out test for GulfResearchAndDevelopment, both BFGS and LBFGS fail on the first iteration.